src/main/resources/db-scripts/cgds.sql

```diff
@@ -45,8 +45,6 @@
 DROP TABLE IF EXISTS `info`;
 DROP TABLE IF EXISTS `clinical_event_data`;
 DROP TABLE IF EXISTS `clinical_event`;
-DROP TABLE IF EXISTS `pdb_uniprot_residue_mapping`;
-DROP TABLE IF EXISTS `pdb_uniprot_alignment`;
```

Comment:
What's this change?

Reply:
These are deprecated tables as far as I know. There is not reason to list them. Or is there? We can omit this change.